assets/js/builder/controls/element/background.js

```diff
@@ -755,8 +757,10 @@ BOLDGRID.EDITOR.CONTROLS = BOLDGRID.EDITOR.CONTROLS || {};
 					name = $this.attr( 'name' ),
 					type = $this.attr( 'data-type' );
 
-				if ( 'class' === type ) {
-					value = BoldgridEditor.colors.defaults[value - 1];
+				if ( 'class' === type && 'neutral' !== value ) {
+					value = 'var(--color-' + BoldgridEditor.colors.defaults[value - 1] + ')';
+				} else if ( 'class' === type && 'neutral' === value ) {
+					value = 'var(--color-neutral)';
```

Comment:
Spacing after and before the parenthesis. You do this later on in the code. Not sure which should be done but wanted to point it out.